<!--xml-->
<launch>


	<include file="$(find ca_rangeflow)/launch/level2_twostage.launch" >
	</include>

	<node pkg="IRONMAP_loopclosure" type="IRONmap" name="IRONmap"  output="screen" args="input:=/camera/depth/points2">
	<param name="sourceFrame" type="string" value="/map"/>
	<param name="targetFrame" type="string" value="/camera_rgb_optical_frame"/>
	<!-- <param name="targetFrame" type="string" value="/openni_rgb_optical_frame" />  -->
        <param name="DownSizeNum" type="int" value="150000" />
        <param name="SkipframeNum" type="int" value="4"/>
        <param name="mapVoxelLeafSize" type="double" value="0.05" />
        <param name="createGlobalMap" type="bool" value="true" />
      <!-- Show parameter in IRON-->
	 <param name="subsamplingFactor" type="double" value="0.2" />
	  <param name="cellSize" type="double" value="0.075" />
          <param name="clippingDistance" type="double" value="4.5" />
	     <param name="neighborSearchRadius" type="double" value="0.6" />
         <param name="entropyThreshold" type="double" value="0.8" />
	     <param name="matchingTolerance" type="double" value="0.1" />
	</node>



</launch>

